src/devices/wirenboard.ts

```diff
@@ -245,22 +254,21 @@ const tzLocal = {
             await entity.read('msRelativeHumidity', ['sprutHeater'], manufacturerOptions);
         },
     } satisfies Tz.Converter,
+    activity_led: {
+        key: ['activity_led'],
+        convertSet: async (entity, key, value, meta) => {
+            const newValue = value === true ? 1 : 0;
+            const options = getOptions(meta.mapped, entity);
+            await entity.write('genBinaryOutput', {presentValue: newValue}, options);
+            return {state: {[key]: value}};
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('genBinaryOutput', ['presentValue']);
+        },
+    } satisfies Tz.Converter,
 };
 
 const sprutModernExtend = {
-    sprutActivityIndicator: (args?: Partial<modernExtend.BinaryArgs>) =>
```

Comment:
Can you debug what goes wrong when using the modern extend? It's the preferred way to do this.

Reply:
When using a modern extend up to version 1.39.0 (inclusive) everything works correctly. But in version 1.39.1, modernExtend.js added the "autoDetectInputEndpoint" function, which leads to an unexpected result:
zh:controller:endpoint: ZCL command 0x7cc6b6fffe5f655b/1 genBinaryOutput.write({"presentValue":1}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false})
and I expect:
zh:controller:endpoint: ZCL command 0x7cc6b6fffe5f655b/5 genBinaryOutput.write({"presentValue":1}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false})
As far as I understand, the command is now sent to endpoint 1 instead of 5. The sensor freezes.
Using the "entity" function as in previous versions fixes this.

Reply:
Could you provide the data/database.db entry of this device?

Reply:
database in case of clean installation of z2m 1.39.1:
[database.db.zip](https://github.com/user-attachments/files/16706252/database.db.zip)

database in the case of proposed fix:
[database.db.zip](https://github.com/user-attachments/files/16706389/database.db.zip)

Reply:
It puzzles me why the code you propose in this PR sends it to ep 5 and not to 1, could you provide the debug log when setting the `activity_led` with your fix included?

See [this](https://www.zigbee2mqtt.io/guide/configuration/logging.html) on how to enable debug logging.

Reply:
[2024-08-23.05-03-32.zip](https://github.com/user-attachments/files/16723744/2024-08-23.05-03-32.zip)

Reply:
I found the issue, attempted to fix it in https://github.com/Koenkk/zigbee-herdsman-converters/pull/7896

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

Reply:
Checked it out, problem's gone. Thank you.